protospec-build/src/lib.rs

```diff
@@ -89,8 +91,13 @@ pub fn rustfmt(input: &str) -> String {
     String::from_utf8_lossy(&proc.wait_with_output().unwrap().stdout).to_string()
 }
 
-pub fn compile_spec(name: &str, spec: &str, options: &Options) -> AsgResult<()> {
-    let resolver = PreludeImportResolver(NullImportResolver);
+pub fn compile_spec<T: ImportResolver + 'static>(
+    name: &str,
+    spec: &str,
+    options: &Options<T>,
+    resolver: T,
+) -> AsgResult<()> {
+    let resolver = PreludeImportResolver(resolver);
```

Comment:
`resolver` is duplicated from within `options`

Reply:
Oversight on my part. I've had to restrict the `T` with `Clone` as well to allow it to be moved.